zc.recipe.egg_/src/zc/recipe/egg/tests.py

```diff
@@ -54,6 +54,8 @@ def test_suite():
                (re.compile('[d-]  setuptools-[^-]+-'), 'setuptools-X-'),
                (re.compile(r'eggs\\\\demo'), 'eggs/demo'),
                (re.compile(r'[a-zA-Z]:\\\\foo\\\\bar'), '/foo/bar'),
+                # control character '\x1b[?1034h' is added on some computers
+                (re.compile(r'\x1b\[\?1034h'), ''),
```

Comment:
Consider adding the link to the Python issue about readline writing these characters to the comment.